Client/src/Components/NavBar/index.jsx

```diff
@@ -46,8 +46,8 @@ function NavBar() {
   };
 
   return (
-    <AppBar position="static">
-      <Container maxWidth="xl">
+    <AppBar position="fixed" sx={{ width: '100%' }}>
+      <Container maxWidth="xl" sx={{ width: '100%' }}>
```

Comment:
does maxWidth and width 100% have any conflict here?  I assume ecplicitly stating the width overrides whatever maxWidth is set to?

If that's the case we can probably just drop the maxWidth prop all together?  That would be less confusing for future developers